aio/scripts/conf.sh

```diff
@@ -38,6 +38,7 @@ BEAUTIFY_BIN="${ROOT_DIR}/node_modules/.bin/js-beautify"
 GLOB_RUN_BIN="${ROOT_DIR}/node_modules/.bin/glob-run"
 GOLINT_URL="https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh"
 GOLINT_BIN="${CACHE_DIR}/golangci-lint"
+GOIMPORTS_BIN="${TMP_DIR}/bin/goimports"
```

Comment:
How did `goimports` get to the temp dir?

Reply:
I agree 👍

Reply:
I copied `goimports` into `.cached_tools` 👌